client/js/models/archive.js

```diff
@@ -72,9 +78,14 @@ module.exports = {
   ],
   effects: {
     new: function (data, state, send, done) {
-      const archive = drive.createArchive(null, {live: true, sparse: true})
+      drive = getDrive()
+      const archive = drive.createArchive(null, {
+        live: true,
+        sparse: true,
+        file: ram
```

Comment:
It seems the ram constructor expects default buffer content, rather than filename to be addressed and access otherwise.  we might need a wrapper around `ram` to emulate filesystem?